tvb/datatypes/local_connectivity.py

```diff
@@ -48,9 +48,9 @@ class LocalConnectivity(HasTraits):
     """
     _ui_name = "Local connectivity"
 
-    surface = Attr(field_type=surfaces.CorticalSurface, label="Surface")
+    surface = Attr(field_type=surfaces.CorticalSurface, label="Surface", required=False)
```

Comment:
This field should remained Required=True, I see in compute_sparse_matrix() method an AttributeError raised, if missing. As we have the validation of required as parts of traits, we should remove that